packages/core/src/lib/lit-core.ts

```diff
@@ -123,7 +130,10 @@ export class LitCore {
   config: LitNodeClientConfigWithDefaults = {
     alertWhenUnauthorized: false,
     debug: true,
-    connectTimeout: 20000,
+    connectTimeout: 20_000,
+    nodeConnectionTimeout: 7_500,
+    nodeRequestTimeout: 35_000, // LitActions have a 30s timeout + some buffer for network latency
+    retryNodeHandshake: true,
```

Comment:
`connectTimeout` is global handshake timeout
`nodeConnectionTimeout` is per node handshake timeout, it overrides `nodeRequestTimeout` at handshake
`nodeRequestTimeout` is per node timeout to apply to ALL requests made to a node
`retryNodeHandshake` is a flag customer can use to allow retrying handshake or failing fast when the first node fails. This will be more handy on naga when we can just discard failing nodes if we reach threshold instead of waiting more to get them